configCivcraft.yml

```diff
@@ -2836,11 +2836,11 @@ recipes:
     input:
       diamond ore:
         material: DIAMOND_ORE
-        amount: 20
+        amount: 32
     output:
       diamond:
         material: DIAMOND
-        amount: 24
+        amount: 64
```

Comment:
Max isn't talking about even competition. It's overwhelming advantage. We're not using exponential outcomes as our balancing metric; it must all be sublinear (so further upgrades yield only marginal increases in outcomes). This fits better as a balancing approach in general.

Let players work out if its worth the investment. High volume producers will recognize the advantage; for others, if they don't, it's a rational decision and one they should make.

In any case this isn't a point we're going to debate here. It's fine if you disagree, but hopefully you can understand the approach we are intentionally taking and respect that PRs attempting to correct issues but violating that general guideline will not be accepted until corrected to somewhere close to those guidelines.